Sources/TuistKit/Commands/Migration/MigrationTargetsByDependenciesCommand.swift

```diff
@@ -0,0 +1,22 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+import TuistSupport
+
+struct MigrationTargetsByDependenciesCommand: ParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(commandName: "list-targets",
+                             abstract: "It lists the targets of a project sorted by number of dependencies.")
+    }
+
+    @Option(
+        name: [.customShort("p"), .long],
+        help: "The path to the Xcode project",
+        completion: .directory
+    )
+    var xcodeprojPath: String
```

Comment:
@andreacipriani what about adding support for workspaces too?

Reply:
I think we can do that in a follow-up PR.

Reply:
Maybe in sub-sequent PR?